python/ray/tune/examples/pbt_memnn_example.py

```diff
@@ -279,7 +279,7 @@ def load_checkpoint(self, path):
         MemNNModel,
         name="pbt_babi_memnn",
         scheduler=pbt,
-        stop={"training_iteration": 10 if args.smoke_test else 100},
+        stop={"training_iteration": 2 if args.smoke_test else 100},
```

Comment:
possible to keep this? I think we wanted to do 2 possible perturbations

Reply:
I had some problems with this test (possibly because of resource constraints), but we might be able to keep this if we increase the timeout (size large or even enormous). Should we update this to 8 though? Shouldn't make a difference for the 25% perturbation rate.

Reply:
I'm trying it out with the latest commit. I had problems with that in earlier rounds, but we'll see how it goes.